limbo/config.py

```diff
@@ -0,0 +1,50 @@
+import os
+from backports import configparser as confparser
+
+
+class Config:
+    env_config = {}
+    file_config = None  # type: confparser.ConfigParser
+
+    def __init__(self):
+        self.load_env_config()
+        self.load_config_file()
+
+    def load_config_file(self):
+        self.file_config = confparser.ConfigParser()
```

Comment:
Why store file config separately from environment config? If we allow that we should accept both, shouldn't they go into the same object?

Reply:
In the future we can easy add option to extensions to write to config file, not just read. 
For that we should keep the ConfigParser object.